docs/specs/clients/notify/notify-authentication.md

```diff
@@ -1,133 +1,150 @@
 # Notify Authentication
 
-In this document we will describe the authentication payloads for different methods:
+In this document we will describe the authentication payloads for all [RPC methods](./rpc-methods.md).
 
-- Notify Subscription
-- Notify Subscription Response
-- Notify Message
-- Notify Receipt
-- Notify Subscription Update
-- Notify Subscription Update Response
-- Notify Subscription Delete
-- Notify Subscription Delete Response
-
-All of the above authentication payloads are DID JWTs and share the following claims:
+All of the authentication payloads are DID JWTs and share the following claims:
 
 - act - description of action intent. Must be equal to specific value defined in each payload
 - iat - timestamp when JWT was issued
 - exp - timestamp when JWT must expire. Must be equal to specific TTL defined in each payload, plus the iat value
 - ksu - key server for identity key verification
```

Comment:
JWTs signed by notify does not need ksu

only clients are registered there

Reply:
Fixed in fe7e2f7793fd166c03feaab9839466f38af7f6cd